include/ensmallen_bits/moead/weight_init_policies/dirichlet_init.hpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file dirichlet_init.hpp
+ * @author Nanubala Gnana Sai
+ *
+ * The Dirichlet method of Weight Initialization.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_MOEAD_DIRICHLET_HPP
+#define ENSMALLEN_MOEAD_DIRICHLET_HPP
+
+namespace ens {
+
+/**
+ * The Dirichlet method for initializing weights. Sampling a dirichlet distribution
+ * with parameters set to one returns points lying on unit simplex with uniform
+ * distribution.
+ *
+ */
```

Comment:
@zoq  Not sure what should be the source ?

Reply:
I don't think there is one we should reference here.